content/identity-service-api.md

```diff
@@ -285,7 +285,7 @@ the client has made an appropriate request to `/hash_details` first.
 ### Security considerations
 
 {{% boxes/note %}}
-[MSC2134](https://github.com/matrix-org/matrix-doc/pull/2134) has much
+[MSC2134](https://github.com/matrix-org/matrix-spec-proposals/pull/2134) has much
```

Comment:
```suggestion
[MSC2134](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/2134-identity-hash-lookup.md) has much
```

since other things are linking to the file rather than the PR?